sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/operation/MetadataOperation.java

```diff
@@ -57,54 +57,6 @@ public void close() throws HiveSQLException {
     cleanupOperationLog();
   }
 
-  /**
```

Comment:
The following method is extracted separately into class `MetadataOperationUtils` and renamed as `legacyXXX`